caller-id-forwarding/functions/forward-call.protected.js

```diff
@@ -0,0 +1,24 @@
+exports.handler = async function (context, event, callback) {
+  const client = context.getTwilioClient();
+
+  const lookup = await client.lookups.v1
```

Comment:
Could this fail if, for example, the call is from an unknown/anonymous number? Arguably that could be a feature (people dialling from anonymous numbers should hear "an application error occurred") but it would collapse the function.

I reckon this should probably be wrapped in a `try/catch` and if it fails forward the call anyway.